src/main/kotlin/day03/Slime.kt

```
@@ -0,0 +1,9 @@
+package org.example.day03
+
+class Slime(var hp: Int, val suffix: String = "A") {
+    val level = 10
+
+    fun attack() {
+
+    }
```

detekt (1.23.7)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)